packages/react-native-renderer/src/ReactFabricGlobalResponderHandler.js

```diff
@@ -7,42 +7,24 @@
  * @flow
  */
 
-// Module provided by RN:
-import {UIManager} from 'react-native/Libraries/ReactPrivate/ReactNativePrivateInterface';
-
 const ReactFabricGlobalResponderHandler = {
   onChange: function (from: any, to: any, blockNativeResponder: boolean) {
-    const fromOrTo = from || to;
-    const fromOrToStateNode = fromOrTo && fromOrTo.stateNode;
-    const isFabric = !!(
-      fromOrToStateNode && fromOrToStateNode.canonical._internalInstanceHandle
-    );
-
-    if (isFabric) {
```

Comment:
is this safe removal because this handler file is solely handling fabric? no mixed case?